### PR TITLE
Feature/contacts

### PR DESCRIPTION
**Pull request by FaithNyamburaWamui**
# Engineers Checklist

## What this PR does?
- List of updates for features, enhancements, or bugfixes

## How can it be tested?
- Please provide step-by-step guide on how to test
- If web or mobile, screenshots will be very helpful in addition

# Where is it documented?
- List JIRA tickets

## Backend Updates
- [ ] Updated Postman Documentation
	- [ ] Requests
	- [ ] Examples
	- [ ] Tests
- [ ] Updated Tasks on JIRA
- [ ] Tested API updates in `dev` environment
- [ ] CI Checks Passed

### CODE REVIEW

**Review by NancyMurangira — DISMISSED** [review]

**Review by Cynthia-mueni — DISMISSED** [review]

**Review by Wangesha-Agnes — DISMISSED** [review]

**Review by Scarlet-Alkeyo — APPROVED** [review]

**Review by Hannah-Unyolo — APPROVED** [review]

File: src/About/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import "./index.css";
 
 const About = () => {
   return (
-    <>
+  <div className="info">
```
Comment (hunterachieng) [inline src/About/index.js:6]: How is about us called info?

File: src/Contact/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+import email from "../image/email.png";
+import location from "../image/location.png";
+import call from "../image/call.png";
+
+
+function Contact() {
+  return (
+    <div className="contact">
+      <div className="header">
```
Comment (hunterachieng) [inline src/Contact/index.js:13]: Why would you place a heading inside a div? Aren't both of them block elements?

**Review by NancyMurangira — DISMISSED** [review]

**Review by NancyMurangira — APPROVED** [review]

**Review by Scarlet-Alkeyo — APPROVED** [review]

**Review by Wangesha-Agnes — APPROVED** [review]

**Review by Cynthia-mueni — APPROVED** [review]

**Review by Hannah-Unyolo — APPROVED** [review]

File: src/About/index.js (hunk 1 above)
Comment (FaithNyamburaWamui) [reply src/About/index.js]: Changed

File: src/Contact/index.js (hunk 2 above)
Comment (FaithNyamburaWamui) [reply src/Contact/index.js]: Changed

File: src/About/index.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import "./index.css";
 
 const About = () => {
   return (
-    <>
+  <div>
       <h3 className="about">ABOUT US</h3>
 
       <div className="line">
```
Comment (hunterachieng) [inline src/About/index.js:9]: What do you mean by line?

Comment (FaithNyamburaWamui) [reply src/About/index.js:9]: It is the class name for the line below the heading of about us.

**Review by hunterachieng — APPROVED** [review]

**Pull request merged**